services/brig/schema/src/Main.hs

```diff
@@ -23,7 +23,6 @@ import qualified V21
 import qualified V22
 import qualified V23
 import qualified V24
-import qualified V25
```

Comment:
is it safe to remove this?

Reply:
Yes, this was the version that migrated the the schema form ascii to text (this isn't allowed in Cassandra 3, which we now use). Since the schema now starts off in text. this is no longer required. There's no other way to bring up this schema in Cassandra 3.